pkg/common/patterns.go

```diff
@@ -0,0 +1,41 @@
+package common
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const EmailPattern = `\b([a-z0-9]{4,25}@[a-zA-Z0-9]{2,12}.[a-zA-Z0-9]{2,6})\b`
```

Comment:
Check out the answer here for the email regex from the latest RFC on it: https://stackoverflow.com/questions/201323/how-can-i-validate-an-email-address-using-a-regular-expression